src/release-scripts/README.md

```diff
@@ -0,0 +1,83 @@
+# Scripts
+
+The following scripts are exposed for use:
+
+## tagAndRelease
+
+This script creates a Git tag and Github release for a given version. It is expected to be run as part of CI and only
+once for the given version defined in `package.json`.
+
+Git tags are created with the following naming scheme: `v1.2.3`. i.e. a `v` is prepended to the version defined in `package.json`.
+
+### Usage
+
+```ts
+// The following environment variable is expected. See the script itself for more details
+//
+//   GH_ACCESS_TOKEN - created through the Github UI with relevant permissions to the repo. S
+
+import { tagAndRelease } from '@api3/commons'
+
+await tagAndRelease('my-repo-name'); // defaults to using the 'main' branch
+await tagAndRelease('my-repo-name', 'release-branch');
```

Comment:
👍 Nice. I guess we can release this and plug it in right away to Auctioneer.

Reply:
Yea thanks for doing this @andreogle! Will apply in https://github.com/api3dao/signed-api/issues/323 and https://github.com/api3dao/airseeker/issues/339